app/api/v1/endpoints/strategies.py

```
@@ -1200,4 +1200,100 @@ async def update_strategy_submission(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail="Failed to update submission"
+        ) from e
+
+
+# User Strategy Management Endpoints
+
+@router.post("/{strategy_id}/toggle")
+async def toggle_user_strategy(
+    strategy_id: str,
+    request: Dict[str, bool],
+    current_user: User = Depends(get_current_user)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)